kde-misc/latte-dock/latte-dock-0.7.2.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
Erm, you aren't changing this file?

Reply:
Yes, shouldn't I?

Reply:
I just changed the year in the file, nothing else.

Reply:
The year should correspond to the last change in the file. So if you aren't changing the file, you should leave the year alone.

Reply:
Is it safe to ignore this warning? 
```
RepoMan scours the neighborhood...
  ebuild.badheader              1
   kde-misc/latte-dock/latte-dock-0.7.2.ebuild: Invalid Gentoo Copyright on line: 1
```